src/neptune/legacy/internal/api_clients/hosted_api_clients/hosted_backend_api_client.py

```diff
@@ -184,33 +182,32 @@ def _verify_version(self):
         parsed_version = version.parse(self.client_lib_version)
 
         if self._client_config.min_compatible_version and self._client_config.min_compatible_version > parsed_version:
-            click.echo(
+            styled_msg = click.style(
```

Comment:
Shouldn't we also call `logger.error(styled_msg)` right after that?

Reply:
this change was removed as its part of legacy client